### ISSUE TITLE
JS error when typing in empty editor that does not at least contain "openapi:"

### ISSUE THREAD

**Issue by jhager** [swagger-api/swagger-editor#1596]
There is a JS error when typing in empty editor that does not at least contain "openapi:" that prevents using the editor. 

Steps to reproduce 

Open new browser window in Incognito mode.  
Go to editor.swagger.io or localhost:3001 
Use File > Clear
Type in "o" and wait one second.

In the browser "😱 Could not render this component, see the console." appear in the browser.  

Error in Javascript console:
swagger-ui.js:1 TypeError: e.get is not a function
    at i (swagger-ui.js:1)
    at swagger-ui.js:1
    at o (swagger-ui.js:1)
    at t.value (topbar.jsx:175)
    at t.B.t.render (swagger-ui.js:1)
    at h._renderValidatedComponentWithoutOwnerOrContext (ReactCompositeComponent.js:796)
    at h._renderValidatedComponent (ReactCompositeComponent.js:819)
    at h._updateRenderedComponent (ReactCompositeComponent.js:743)
    at h._performComponentUpdate (ReactCompositeComponent.js:721)
    at h.updateComponent (ReactCompositeComponent.js:642)

Workaround:  Paste in "openapi:" or type faster

| Q                                   | A
| ----------------------------------- | -------
| Bug or feature request?             |   Bug
| Which Swagger/OpenAPI version?      | Not Applicable
| Which Swagger-Editor version?       |. v3.1.20

Bug exist in:  commit 4c7b4b98c705e24af6eda109d84a8bc7e684dc26 (HEAD -> master, tag: v3.1.20, origin/master, origin/HEAD)

It is not reproducible in: commit 08776a9e42a7fd714a1a1d16d1b5c8702fce91f7 (tag: v3.1.19)

| How did you install Swagger-Editor? |   download from github.  Can also reproduce on http://editor.swagger.io (as of Dec 19th)
| Which broswer & version?            |  Safari or Chrome
| Which operating system?             |  Mac OS X.

### Expected Behavior

Should still be able to use the editor when slowly typing the first line.

**Comment by reginald-louis** [comment]
I can also add that once it happen, unless you delete the local cookie, it's impossible to use the editor!

**Comment by cthain** [comment]
I have exactly the same problem with the swagger-editor tag v3.1.20. I have reproduced this consistently on Windows 7/Chrome 63.0.3239.84 and OpenSUSE Leap 42.3/Firefox 52.5.2 following the steps described by the OP. This is a pretty significant error and after it happens the only way to restore the editor functionality is to clear the browser cache (cookies and cached data). I have also observed that it _doesn't_ happen with tag v3.1.19.
